### PR TITLE
Fix broken link

### PR DESCRIPTION
**Pull request by wombatonfire**
Link to "Azure AD Connect: Design concepts" leads to 404.

### CODE REVIEW

**Comment by wombatonfire** [comment]
I guess there should not be a direct link to docs.microsoft.com, but the redirection link is broken and doesn't lead to https://docs.microsoft.com/en-us/azure/active-directory/hybrid/plan-connect-design-concepts

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Skype/SfbHybrid/hybrid/configure-a-multi-forest-environment-for-hybrid.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ To get proper identity synchronization, the following attributes need to be sync
 |ProxyAddresses  <br/> |ProxyAddresses  <br/> |
 |ObjectSID  <br/> |msRTCSIP-OriginatorSID  <br/> |
    
-The [chosen account link attribute](https://azure.microsoft.com/documentation/articles/active-directory-aadconnect-design-concepts/) will be used as the Source Anchor. If you have a different and immutable attribute that you would prefer to use, you may do so; just be sure to edit the AD FS claims rule and select the attribute during the AAD Connect configuration.
+The [chosen account link attribute](https://docs.microsoft.com/en-us/azure/active-directory/hybrid/plan-connect-design-concepts) will be used as the Source Anchor. If you have a different and immutable attribute that you would prefer to use, you may do so; just be sure to edit the AD FS claims rule and select the attribute during the AAD Connect configuration.
```
Comment (JohanFreelancer9) [inline Skype/SfbHybrid/hybrid/configure-a-multi-forest-environment-for-hybrid.md:63]:
```suggestion
The [chosen account link attribute](https://docs.microsoft.com/azure/active-directory/hybrid/plan-connect-design-concepts) will be used as the Source Anchor. If you have a different and immutable attribute that you would prefer to use, you may do so; just be sure to edit the AD FS claims rule and select the attribute during the AAD Connect configuration.
```

**Comment by scanum** [comment]
@CarolynRowe PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**